Gemfile

```diff
@@ -50,6 +50,9 @@ gem 'graphiql-rails'
 # Pundit
 gem 'pundit'
 
+gem 'exception_notification'
+gem 'slack-notifier'
```

Comment:
We want to run this in production and all other environments?

Reply:
According to this https://github.com/smartinez87/exception_notification#rails-1 - it should be installed in all environments, I already have a setting in the config to only send notifications on production. In any case it won't send any notifications if you don't set the webhook.